dirsrvtests/tests/suites/clu/dsidm_account_test.py

```diff
@@ -1,21 +1,30 @@
 # --- BEGIN COPYRIGHT BLOCK ---
-# Copyright (C) 2023 Red Hat, Inc.
+# Copyright (C) 2025 Red Hat, Inc.
 # All rights reserved.
 #
 # License: GPL (version 3 or any later version).
 # See LICENSE for details.
 # --- END COPYRIGHT BLOCK ---
 #
-import ldap
-import time
-import subprocess
-import pytest
 import logging
 import os
-
+import json
+import pytest
+import ldap
 from lib389 import DEFAULT_SUFFIX
-from lib389.cli_idm.account import list, get_dn, lock, unlock, delete, modify, rename, entry_status, \
-    subtree_status, reset_password, change_password
+from lib389.cli_idm.account import (
+    # list,
+    get_dn,
+    lock,
+    unlock,
+    # delete,
+    # modify,
+    # rename,
+    entry_status,
+    subtree_status,
+    # reset_password,
+    # change_password
```

Comment:
If we don't need imports, I think it's better just remove them

Reply:
> If we don't need imports, I think it's better just remove them

I guess I saw this as a todo list :-)